plugins/kubernetes-backend/src/cluster-locator/ConfigClusterLocator.ts

```diff
@@ -50,6 +51,9 @@ export class ConfigClusterLocator implements KubernetesClustersSupplier {
         if (c.has('dashboardParameters')) {
           clusterDetails.dashboardParameters = c.get('dashboardParameters');
         }
+        if (c.has('displayName')) {
+          clusterDetails.displayName = c.get('displayName');
+        }
```

Comment:
why is this needed? You already set it in line 41. You can remove 54-56.

line 51-53 could be simplified, too

Reply:
@pjungermann if I slip this condition, the `displayName: undefined` will be passed to every `clusterDetails` object. So we will get something like
```
{
    name: 'other-cluster',
    displayName: undefined,
    authProvider: 'google',
}
```

I would keep it as is. The same applied for KubernetesFanOutHandler

Reply:
hmm.. it is anyways a property which defaults to `undefined` but ok 🤔